db/rdbms-schema/src/main/resources/db/changelog/rdbms-exporter/changesets/8.8.0.xml

```diff
@@ -343,10 +343,10 @@
 
   <changeSet id="create_tenant_table" author="cthiel">
     <createTable tableName="TENANT">
-      <column name="TENANT_KEY" type="BIGINT">
+      <column name="TENANT_ID" type="VARCHAR(255)">
         <constraints primaryKey="true"/>
       </column>
-      <column name="TENANT_ID" type="VARCHAR(255)" />
+      <column name="TENANT_KEY" type="BIGINT"/>
```

Comment:
:question: Is this still being used anywhere?

Reply:
Good question, there is still the concept of a `tenant key` for the memberships (I am planning to follow up with a separate PR about that), for this field on the table though its likely something that we can remove as part of that clean up.